scheduler/pkg/envoy/xdscache/seldoncache_test.go

```diff
@@ -38,9 +39,14 @@ func TestAddRemoveHttpAndGrpcRoute(t *testing.T) {
 	addVersionedRoute(c, route1, model1, httpCluster, grpcCluster, 100, 1)
 	addVersionedRoute(c, route2, model1, httpCluster, grpcCluster, 100, 1)
```

Comment:
question: how can we have 2 routes pointing to the same cluster in the current logic?

Reply:
When a model is part of an experiment.